### PR TITLE
DOCSP-20688 additional mongostat examples

### PR DESCRIPTION
**Pull request by davemungo**
### STAGING
https://docs-mongodbcom-staging.corp.mongodb.com/database-tools/docsworker-xlarge/DOCSP-20688-additional-mongostat-examples/mongostat/#std-option-mongostat.-o

https://docs-mongodbcom-staging.corp.mongodb.com/database-tools/docsworker-xlarge/DOCSP-20688-additional-mongostat-examples/mongostat/#add-fields-to--output

### JIRA
https://jira.mongodb.org/browse/DOCSP-20688
[TOOLS] include additional examples for mongostat

### CODE REVIEW

File: source/mongostat.txt [hunk 1]
```diff
@@ -401,15 +401,16 @@ Options
 
 .. option:: -o=<field list>
 
-   When specified, :program:`mongostat` includes **only** the specified fields
-   in the :program:`mongostat` output.
+   When specified, :program:`mongostat` includes **only** the specified
```
Comment (davemungo) [inline source/mongostat.txt:404]: The initial group of changes, from lines 404 - 465, include a lot of line-length fixes.

The content updates for that section are on lines:
411-13
460-62

**Review by davemungo — COMMENTED** [review]
Line wrap comment

**Review by jason-price-mongodb — APPROVED** [review]
@davemungo LGTM with suggestions. Cheers.

File: source/mongostat.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (davemungo) [reply source/mongostat.txt]: Done

File: source/mongostat.txt [hunk 2]
```diff
@@ -721,18 +725,26 @@ behavior:
...
-:option:`-O <mongostat -O>` allows you to specify fields from
+:option:`-O <mongostat -O>` allows you to specify fields from the
 :dbcommand:`serverStatus` output to add to the default
-:binary:`~bin.mongostat` output. The following example adds the :serverstatus:`host`
-and :serverstatus:`version` fields as well as the :serverstatus:`network.numRequests` field,
-which will display as "network requests", to the default
+:binary:`~bin.mongostat` output. If you include a space in your
+custom filed name, do not put additional quotes around the field name. 
+
+The following example adds three fields to the default
```
[the remark this answers is not in the document]
Comment (davemungo) [reply source/mongostat.txt]: Done

File: source/mongostat.txt [hunk 3]
```diff
@@ -721,18 +725,26 @@ behavior:
...
 :binary:`~bin.mongostat` output:
 
+- :serverstatus:`host`
+- :serverstatus:`version`
+- :serverstatus:`network.numRequests`
+
+The :serverstatus:`network.numRequests` field has a custom field name,
```
[the remark this answers is not in the document]
Comment (davemungo) [reply source/mongostat.txt]: Done

**Review by davemungo — COMMENTED** [review]
Thanks for the feedback @jason-price-mongodb sending on to tech review next

**Comment by davemungo** [comment]
@irenazaidman 

This is the mongostats docs update for field names with spaces. Would you have a look?

**Pull request merged**